src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -0,0 +1,594 @@
+                  mt={4}
+                >
+                  <Stack direction='row'>
+                    <Radio
+                      id='POCisAuthorisedSignatory-yes'
+                      size='lg'
+                      name='POCisAuthorisedSignatory'
+                      value='Yes'
+                      defaultChecked
+                      mr={8}
+                    >
+                      <PageText fontSize='input'>Yes</PageText>
+                    </Radio>
+
+                    <Radio
+                      id='POCisAuthorisedSignatory-no'
+                      size='lg'
+                      name='POCisAuthorisedSignatory'
+                      value='No'
+                    >
+                      <PageText fontSize='input'>No</PageText>
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+              </Field>
+            )}
+          />
+
+          <Box display={notAuthorisedSignatory ? 'block' : 'none'}>
+            <Fade in={notAuthorisedSignatory} delay={0.25}>
+              <TextField
+                id='authorisedSignatoryInformation'
+                label='Name, job title, and email address of the authorised signatory'
+                helpText='(e.g. John Smith, CEO, john@mycompany.com. This is the person who will sign the contract. They must be someone who can sign contracts on behalf of the entity)'
+                isRequired
+              />
+            </Fade>
+          </Box>
+
+          <Controller
+            name='applyingAs'
+            control={control}
+            render={({ field: { onChange } }) => (
+              <Field id='applyingAs' label='In which capacity are you applying?' isRequired>
+                <Select
+                  id='applyingAs'
+                  options={APPLYING_AS_OPTIONS}
+                  onChange={option =>
+                    onChange((option as typeof APPLYING_AS_OPTIONS[number]).value)
+                  }
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+              </Field>
+            )}
+          />
+
+          <Box display={applyingAs === OTHER ? 'block' : 'none'}>
+            <Fade in={applyingAs === OTHER} delay={0.25}>
+              <TextField id='applyingAsOther' label='If other, please specify' />
+            </Fade>
+          </Box>
+
+          <TextField
+            id='company'
+            label='Institution or Organization'
+            helpText='Name of your university program, team, or organization. If you do not have an organization name, write "N/A"'
+            isRequired
+          />
+
+          <Controller
+            name='country'
+            control={control}
+            render={({ field: { onChange }, fieldState: { error } }) => (
+              <Field
```

Comment:
Select missing the error PageText when no option is selected

Reply:
This one is showing the error already. Can you double check? I tested and I see the error.